specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json

```diff
@@ -537,14 +541,14 @@
         "allOf": {
           "type": "array",
           "items": {
-            "$ref": "#/definitions/MetricCriteria"
+            "$ref": "#/definitions/StaticMetricCriteria"
```

Comment:
Breaking change in stable API.
Any reason you would like to change your models in a stable version?

Reply:
Fixed, please see updates.